packages/sdk/src/internal/External/StakeWiseV3.ts

```diff
@@ -0,0 +1,34 @@
+import { Viem } from "@enzymefinance/sdk/Utils";
+import { type Address, type PublicClient, parseAbi } from "viem";
+
+export async function getSharesBalance(
+  client: PublicClient,
+  args: Viem.ContractCallParameters<{
+    account: Address;
+    stakeWiseVaultAddress: Address;
+  }>,
+) {
+  return Viem.readContract(client, args, {
+    abi: parseAbi(["function balanceOf(address _account) view returns (uint256 balance_)"]),
+    functionName: "balanceOf",
+    address: args.stakeWiseVaultAddress,
+    args: [args.account],
+  });
+}
```

Comment:
You don't need this, you can just use `Assets.getBalanceOf()`

Reply:
thanks, removed